special-pages/pages/new-tab/app/omnibar/components/TabSwitcher.module.css

```diff
@@ -53,6 +54,7 @@
     translate: calc(2px + var(--tab-index) * (100% + 2px));
     width: calc((100% - 4px - (var(--tab-count) - 1) * 2px) / var(--tab-count));
     will-change: translate;
+    z-index: 0;
```

Comment:
Not necessary, just adding it to make it explicit that `.blob` is supposed to be under `.tab`.